tests/src/test_cases/libs/test_libwebp.c

```diff
@@ -0,0 +1,72 @@
+#if LV_BUILD_TEST
+#include "../lvgl.h"
+#include "../../lvgl_private.h"
+
+#include "unity/unity.h"
+
+void setUp(void)
+{
+    /* Function run before every test */
+}
+
+void tearDown(void)
+{
+    /* Function run after every test */
+}
+
+static void create_image_item(lv_obj_t * parent, const void * src, const char * text)
+{
+    lv_obj_t * cont = lv_obj_create(parent);
+    lv_obj_remove_style_all(cont);
+    lv_obj_set_size(cont, 300, 200);
+    lv_obj_set_flex_flow(cont, LV_FLEX_FLOW_COLUMN);
+    lv_obj_set_flex_align(cont, LV_FLEX_ALIGN_CENTER, LV_FLEX_ALIGN_CENTER, LV_FLEX_ALIGN_CENTER);
+
+    lv_obj_t * img = lv_image_create(cont);
+    lv_image_set_src(img, src);
+
+    lv_obj_t * label = lv_label_create(cont);
+    lv_label_set_text(label, text);
+}
+
+static void create_webp_images(void)
```

Comment:
There are no webp test cases without extensions.

Reply:
Prevent regression issues caused by the judgment of file extensions added by someone in the future